cpp/include/raft/neighbors/ivf_pq_helpers.cuh

```diff
@@ -407,5 +575,160 @@ void erase_list(raft::resources const& res, index<IdxT>* index, uint32_t label)
   ivf_pq::detail::erase_list(res, index, label);
 }
 
+/**
+ * @brief Public helper API exposing the computation of the index's rotation matrix.
+ * NB: This is to be used only when the rotation matrix is not already computed through
+ * raft::neighbors::ivf_pq::build.
+ *
+ * Usage example:
+ * @code{.cpp}
```

Comment:
Please expand this example. This is meant to be copy/paste for users, not a simple copy of the function prototype into the docs.

Reply:
Done